### PR TITLE
MS. Don. e. 250: new description for studio archive migration project

### PR DESCRIPTION
**Pull request by holfordm**

### CODE REVIEW

File: collections/Don/MS_Don_e_250.xml [hunk 1]
```diff
@@ -71,10 +76,441 @@
...
+                        <rubric>Ad primum completorium</rubric>
+                        <incipit>Domine ihesu criste fili dei viui qui hora completorij pro me misero peccatore imminente passione tua tristarj pauere et mestus esse voluisti</incipit>
+                        <explicit>et ego tibi consepultus peruenire tecum merear  ad resurrectionis gloriam. Amen</explicit></msItem>
+                     <msItem n="6" xml:id="MS_Don_e_250-partA-item3">
+                        <locus>(fols 32v–34v)</locus>
+                        <title type="desc" key="work_13950">Prayers</title>
+                        <msItem>
+                           <locus>(fol. 32v–33r)</locus>
+                           <rubric> Oracio ad wlnera cristi</rubric>
+                           <incipit>Exaltabo te domine qui suscepisti me  O Solutifera wlnera dulcissimi  amatoris mei ihesu cristi</incipit>
```
Comment (adunning) [inline collections/Don/MS_Don_e_250.xml:191]: Are these extra spaces meant to represent line breaks?

File: collections/Don/MS_Don_e_250.xml [hunk 2]
```diff
@@ -71,10 +76,441 @@
...
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem n="7" xml:id="MS_Don_e_250-partBi-item7">
+                        <locus>(fols 35r–36v)</locus>
+                        <rubric> Modus Meditandi ihesus Memoriale passionis dominice</rubric>
+                        <incipit>Sedule mediteris eius passionem et infigar cordi tuo sicut ipse iugiter licet et fert infixa wlnerum</incipit>
+                        <explicit>graciarum plenitudinem / et a cunctis  malis imminentibus securam protectionem</explicit>
+                     </msItem>
+                     <msItem n="8" xml:id="MS_Don_e_250-partB-item8">
+                        <locus>(fols 36v–38r)</locus><!-- recte 39r? -->
+                        <author key="person_4949605"><supplied>LUDOLPH OF SAXONY</supplied></author>
```
Comment (adunning) [inline collections/Don/MS_Don_e_250.xml:265]: I am not clear on why names are in all caps.

File: collections/Don/MS_Don_e_250.xml [hunk 3]
```diff
@@ -71,10 +76,441 @@
...
+                        <title key="work_3016"><supplied>Vita Jesu Christi</supplied></title>
```
Comment (adunning) [inline collections/Don/MS_Don_e_250.xml:266]: Are you wanting to experiment with using `<supplied>` with `<author>` and `<title>`, or is this left over from the original description?

File: collections/Don/MS_Don_e_250.xml [hunk 4]
```diff
@@ -71,10 +76,441 @@
...
+                           <country key="place_7000084">German</country>, <region>west</region> (?)
+                        </origPlace>
+                     </origin>
+                  </history>
+               </msPart>
             </msDesc>
          </sourceDesc>
       </fileDesc>
       <revisionDesc>
+         <change when="2021-09">New description encoded from description by Jonas Hermann.</change>
```
Comment (adunning) [inline collections/Don/MS_Don_e_250.xml:513]: Your name is missing.

Comment (holfordm) [reply collections/Don/MS_Don_e_250.xml:513]: agree that the encoder's name should be given here if it is not already given in `source` - but isn't it redundant if already in `source`?

File: collections/Don/MS_Don_e_250.xml (hunk 3 above)
Comment (holfordm) [reply collections/Don/MS_Don_e_250.xml:266]: convention in the original description - common convention in descriptions from the German speaking world and also in descriptions of Greek mss. in my experience. It has the virtue of making explicit whether an author/title is explicitly named in the rubric.

File: collections/Don/MS_Don_e_250.xml (hunk 2 above)
Comment (holfordm) [reply collections/Don/MS_Don_e_250.xml:265]: left over from original description

File: collections/Don/MS_Don_e_250.xml (hunk 1 above)
Comment (holfordm) [reply collections/Don/MS_Don_e_250.xml:191]: They may represent places where I removed existing line breaks (where the description was recording each new line of the manuscript)

**Pull request merged**